src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -176,8 +176,6 @@ def parse_performance_features(log_lines, strategies, arguments):
       'max_len': 0,
       'manual_dict_size': 0,
       'merge_edge_coverage': 0,
-      'new_edges': 0,
```

Comment:
Why remove the default values? I suspect that we may end up with `null` values in BigQuery, do we want that?

Reply:
Yes it helps to calculate averages correctly for graphs. we used to use similar strategy for corpus_rss_mb and i think it used to work.

Reply:
Got it, thanks!